components/molecules/NavbarDashboard/index.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import Link from 'next/link';
 
 import { Icon } from 'antd';
 import { NavBarDiv } from './index.styled';
 import { Logo } from '../../atoms/Logo';
+import { Button } from '../../atoms/Button';
 
 const NavBarDashboard = () => (
```

Comment:
We will need to discard this component and refactor the NavBar molecule for this in future updates.